recipes-multimedia/aamp/aamp_git.bb

```diff
@@ -7,7 +7,7 @@ LIC_FILES_CHKSUM = "file://LICENSE;md5=175792518e4ac015ab6696d16c4f607e"
 # extra DEPENDS
 DEPENDS_append = " \
     curl libdash libxml2 cjson aampabr aampmetrics wpeframework \
-    gstreamer1.0 gstreamer1.0-plugins-base glib-2.0 util-linux wpeframework-clientlibraries \
+    gstreamer1.0 gstreamer1.0-plugins-base glib-2.0 util-linux wpeframework-clientlibraries wpewebkit \
```

Comment:
Does this work from Streamer plugin as like earlier?

Reply:
@HaseenaSainul, Does the Streamer needs something from wpewebkit for AAMP to work? If not then it should be on the PACKAGECONFIG